crowbar_framework/config/initializers/delayed_job_config.rb

```diff
@@ -14,12 +14,27 @@
 # limitations under the License.
 #
 
+module Delayed
+  class Worker
+    @@log_file = if Rails.env.production?
+      File.join(ENV["CROWBAR_LOG_DIR"], "background_jobs.log")
+    else
+      File.join(Rails.root, "log", "background_jobs.log")
+    end
+    @@logger = Logger.new(@@log_file)
```

Comment:
Why you don't keep the old way? I don't understand this change and the code looks unclean. Please explain.